### ISSUE TITLE
Swagger: NotSupportedException when using EndpintWithoutRequest

### ISSUE THREAD

**Issue by gebirgslok** [FastEndpoints/FastEndpoints#602]
The following Endpoint worked totally fine up to and including 5.21.0:
`public sealed class MyEndpoint : EndpointWithoutRequest<MyDto> { ... }`

Howevery, since 5.22.0, I receive the following exception when trying to access /swagger:

`Request DTOs without any publicly accessible properties are not supported. Offending Endpoint: [MyEndpoint] Offendi
ng DTO type: [FastEndpoints.EmptyRequest]`

I guess that after the "Warn user about illegal request DTO types" change, EmptyRequest violates this rule as well?

BR
Jens

**Comment by dj-nitehawk** [comment]
[just tried the following](https://streamable.com/qpe0uh) and it works just fine:
```cs
var bld = WebApplication.CreateBuilder(args);
bld.Services
   .AddFastEndpoints()
   .SwaggerDocument();

var app = bld.Build();
app.UseFastEndpoints()
   .UseSwaggerGen();
app.Run();

sealed class MyDto
{
    public string Msg { get; set; }
}

sealed class MyEndpoint : EndpointWithoutRequest<MyDto>
{
    public override void Configure()
    {
        Post("test");
        AllowAnonymous();
    }

    public override async Task HandleAsync(CancellationToken c)
    {
        await SendAsync(new() { Msg = "hello" });
    }
}
```
can you show me how to reproduce it?
or attach a repro project.

possible duplicate of #599

**Comment by gebirgslok** [comment]
Ok, I figured out the reason that broke it:

`
sealed class MyEndpoint : EndpointWithoutRequest<MyDto>
{
    public override void Configure()
    {
        Post("test");
        AllowFileUploads(); // This causes the exception   
        AllowAnonymous();
    }

    public override async Task HandleAsync(CancellationToken c)
    {
       var filesCount = Files.Count;
        await SendAsync(new() { Msg = "hello" });
    }
}
`

**Comment by dj-nitehawk** [comment]
thank you!
this has now been fixed in `v5.22.0.2-beta`.
your example above will no longer throw.
